zebra-state/src/service/finalized_state.rs

```diff
@@ -41,8 +41,21 @@ impl FinalizedState {
             rocksdb::ColumnFamilyDescriptor::new("sprout_nullifiers", db_options.clone()),
             rocksdb::ColumnFamilyDescriptor::new("sapling_nullifiers", db_options.clone()),
         ];
-        let db = rocksdb::DB::open_cf_descriptors(&db_options, path, column_families)
-            .expect("database path and options are valid");
+        let db_result = rocksdb::DB::open_cf_descriptors(&db_options, &path, column_families);
+
+        let db = match db_result {
+            Ok(d) => {
+                tracing::info!("Opened Zebra state cache at: {:?}", path);
```

Comment:
The tracing macros let us provide variables as separate data, rather than putting them all in the same string. If we ever want to filter or process tracing data, it's much easier if variables are separate data.

https://tracing.rs/tracing/#recording-fields

```suggestion
                tracing::info!(?path, "Opened Zebra state cache");
```

Reply:
Actually, we can't make this change, because the tests depend on the exact format of the message.

But let's be consistent with the port messages by deleting the extra `:`.

```suggestion
                tracing::info!("Opened Zebra state cache at {:?}", path);
```